JuliaExperimental/julia/gaprat.jl

```diff
@@ -12,8 +12,6 @@ import Base: zero, -, one, inv, ==, isless, +, *, //, ^, mod, iszero, string,
 
 import Main.GAP: GapObj
 
-import Main.GAP.GAPFuncs: SUM
-
```

Comment:
I had to add this to make it possible to load this file (good thing we fixed the tests :-)